src/transports/dash/text_parser.ts

```diff
@@ -63,11 +63,39 @@ function parseMP4EmbeddedTrack({ response,
                                                                        0);
 
   if (isInit) {
-    const mdhdTimescale = getMDHDTimescale(chunkBytes);
-    const chunkInfos = mdhdTimescale > 0 ? { time: 0,
-                                             duration: 0,
-                                             timescale: mdhdTimescale } :
-                                           null;
+    const { privateInfos } = segment;
+    const shouldExtractCompleteInitChunk = privateInfos?.shouldGuessInitRange === true;
+
+    const completeInitChunk = shouldExtractCompleteInitChunk ?
+      extractCompleteInitChunk(chunkBytes) : chunkBytes;
+
+    if (completeInitChunk === null &&
+      (
+        sidxSegments === null ||
+        sidxSegments.length === 0
+      ) &&
+      privateInfos?.shouldGuessInitRange === true &&
+      segment.indexRange === undefined
+    ) {
```

Comment:
Same here

Reply:
We still have to check for next segments presence. If the complete init chunk is null and the content may be static, we still can find a sidx and parse new segments from it.

Reply:
Re-same here